Source/WebKit/UIProcess/Automation/WebDriverBidiProcessor.cpp

```diff
@@ -111,19 +117,177 @@ void WebDriverBidiProcessor::sendMessageToFrontend(const String& message)
 }
 
 
-// MARK: Inspector::BrowsingContextDispatcherHandler methods.
+// MARK: Inspector::BidiBrowsingContextDispatcherHandler methods.
 
-void WebDriverBidiProcessor::navigate(const BrowsingContext& browsingContext, const String& url, std::optional<Inspector::Protocol::BidiBrowsingContext::ReadinessState>&&, CommandCallbackOf<String, String>&& callback)
+void WebDriverBidiProcessor::activate(const BrowsingContext& browsingContext, CommandCallback<void>&& callback)
 {
     RefPtr session = m_session.get();
     if (!session)
         ASYNC_FAIL_WITH_PREDEFINED_ERROR(InternalError);
 
-    ASYNC_FAIL_WITH_PREDEFINED_ERROR(NotImplemented);
+    // FIXME: implement `frameHandle` to activate non-top level browsing contexts.
```

Comment:
Per the spec, we should return `"invalid argument"` when trying to activate non-top level browsing contexts.

Step 3 in https://w3c.github.io/webdriver-bidi/#command-browsingContext-activate : 
> If navigable is not a [top-level traversable](https://html.spec.whatwg.org/multipage/document-sequences.html#top-level-traversable), return [error](https://w3c.github.io/webdriver/#errors) with [error code](https://w3c.github.io/webdriver/#dfn-error-code) [invalid argument](https://w3c.github.io/webdriver/#dfn-invalid-argument).
```suggestion
    // FIXME: detect non-top level browsing contexts, returning `invalid argument`
```